### PR TITLE
8/28/18 SOFT SYNC TO TGSTATION LATEST

### PR DESCRIPTION
**Pull request by silicons**

### CODE REVIEW

File: code/datums/diseases/advance/symptoms/nanites.dm [hunk 1]
```diff
@@ -0,0 +1,63 @@
+/datum/symptom/nano_boost
```
Comment (deathride58) [inline code/datums/diseases/advance/symptoms/nanites.dm:1]: File will cause conflicts with nanites port

**Comment by deathride58** [comment]
does this include BSQL or nah

**Comment by deathride58** [comment]
Ping me whenever this is ready for merge

**Comment by Poojawa** [comment]
I'm getting the feeling I should do the sync, because I can get it done and finished within an hour or two

**Comment by deathride58** [comment]
Fix the conflicts and fix the compiling errors already

**Comment by silicons** [comment]
going to need like a 8 hour block of time to do this right

**Pull request closed without merging**